# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../prjtrellis/python37.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libtrellis/CMakeLists.txt.vanilla	2020-02-16 12:28:37.353238395 +0100
+++ libtrellis/CMakeLists.txt	2020-02-16 12:28:45.149238301 +0100
@@ -57,7 +57,7 @@
     endif ()
 
     if (NOT Boost_PYTHON_FOUND)
-        find_package(Boost COMPONENTS python36 ${boost_libs})
+        find_package(Boost COMPONENTS python37 ${boost_libs})
         if ("${Boost_LIBRARIES}" MATCHES ".*(python|PYTHON).*" )
             set(Boost_PYTHON_FOUND TRUE)
         endif ()
